app/hooks/android_adjust_soft_keyboard/index.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import RNUtils from '@mattermost/rnutils';
+import {useEffect, useRef} from 'react';
+import {Keyboard} from 'react-native';
+import {Navigation} from 'react-native-navigation';
+
+import type {AvailableScreens} from '@typings/screens/navigation';
+
+export function useAndroidAdjustSoftKeyboard(screen?: AvailableScreens) {
+    const timeout = useRef<NodeJS.Timeout>();
+
+    useEffect(() => {
+        const listener = {
+            componentDidAppear: () => {
+                if (Keyboard.isVisible()) {
+                    Keyboard.dismiss();
+                }
+                timeout.current = setTimeout(() => {
+                    RNUtils.setSoftKeyboardToAdjustNothing();
+                }, 10);
```

Comment:
Probably is never going to happen, and the timeout being 10ms is not going to make any real difference, but I ask just in case.
If we have several repeated events for componentDidAppear, we may create several timeouts that will do the same (`setSoftKeyboardToAdjustNothing`). Is that ok?

Reply:
Why would componentDidAppear fire repeatedly?

Reply:
I don't know. I guess it won't. But I preferred to ask just in case :stuck_out_tongue: